### PR TITLE
[community] Member profile : Divyangi Raghav

### LINKED ISSUE #3913: [Community] Member Profile: Divyangi Raghav
**Current Behavior**

@divz-codess has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @divz-codess as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @divz-codess
- Twitter: 
- LinkedIn: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by divz-codess**
**Description**

This PR fixes #3913 

**Notes for Reviewers**
Created a community page for @divz-codess 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 818cac14401902aa1f9296ce8eb592c1090b41ed at: https://64083c620e2b2273f52195f3--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 0d0bd4d48053ce107f89b259a7937f8d8d0716ef at: https://6408d68766f09a7ca3e81a85--layer5.netlify.app

File: src/collections/members/divyangi-raghav/index.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+name: Divyangi Raghav 
+position: Community Manager
+image_path: ./divyangi-raghav.jpg
+github: divz-codess
+twitter: DivzzTwt
+linkedin: divyangi-raghav
+badges:
+ - layer5
+ - meshery
+ - smp
```
Comment (leecalcote) [inline src/collections/members/divyangi-raghav/index.mdx:11]:
```suggestion
 - community
```

Comment (leecalcote) [reply src/collections/members/divyangi-raghav/index.mdx:11]: Psst. Community Manager @divz-codess, read these - https://layer5.io/community/handbook/recognition#Badges and Community Manager Trainer @Chadha93, ensure this is included as required reading.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/collections/members/divyangi-raghav/index.mdx (hunk 1 above)
Comment (divz-codess) [reply src/collections/members/divyangi-raghav/index.mdx]: Done @leecalcote

Comment (Chadha93) [reply src/collections/members/divyangi-raghav/index.mdx]: Sure, @leecalcote

Comment (Chadha93) [reply src/collections/members/divyangi-raghav/index.mdx]: @divz-codess 
Please remove these badges:
 - layer5
 - meshery
 - smp
 
and let's update the position - community manager after the official announcement. You can add it as Contributor.

Comment (Chadha93) [reply src/collections/members/divyangi-raghav/index.mdx]: Yes, take reference: https://github.com/layer5io/layer5/pull/3925

**Comment by l5io** [comment]
🚀 Preview for commit 9c143abd8d22eb4ec5791f38b958a6ed3933a5e3 at: https://640aef119d1e9468bc6f832f--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4190488ff5e051fbf78766b7731557ef857a4d50 at: https://640b9286b6873902261faec8--layer5.netlify.app

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**